src/dashboard/src/main/management/commands/__init__.py

```diff
@@ -0,0 +1,22 @@
+from django.core.management.base import BaseCommand
+from django.utils.six.moves import input
+
+
+class DashboardCommand(BaseCommand):
+    def success(self, message):
+        self.stdout.write(self.style.MIGRATE_SUCCESS(message))
+
+    def error(self, message):
+        self.stdout.write(self.style.ERROR(message))
+
+    def warning(self, message):
+        self.stdout.write(self.style.WARNING(message))
+
+
+def boolean_input(question, default=None):
+    result = input('%s ' % question)
+    if not result and default is not None:
+        return default
+    while len(result) < 1 or result[0].lower() not in 'yn':
+        result = input('Please answer yes or no: ')
+    return result[0].lower() == "y"
```

Comment:
Kinda weird that I could enter "yno" here and this would be interpreted as "yes".